### DISCUSSION TITLE
Assumptions in tutorial, matching footprint & inventory resolutions, and summation vs integration

### DISCUSSION THREAD

**Discussion by Griffin-Mead** [uataq/stilt#124] [Users]
Hi,

I'm interested in comparing point sensor methane measurements to local emissions inventories by scaling the inventories with STILT-R footprints. This is motivated by the WBB Carbon Dioxide Github example where observed enhancements were estimated as:

`dCO2 = sum(values(foot * emissions_subset), na.rm = T))`

A series of questions have arisen while trying to apply this approach to my work.

First, the tutorial mentions that this approach "makes some overly-basic assumptions" - could you elaborate on what these assumptions are and when they do not hold?

Second, what is the proper way to change the simulation's spatial resolution when using the summation approach? For instance, the WBB example is run at 0.002 deg^2 (~222 m^2) resolution. Footprint and emissions units are ppm m^2 s umol^-1 and umol m^-2 s^-1 respectively. A reasonable dCO2 enhancement is calculated with this combination of spatial resolutions/units. However, the summation approach also means that changing the spatial resolution dramatically changes the estimated enhancement (more/fewer points increases/decreases the estimated enhancement). Do the footprint and emissions units need to be rescaled relative to the 0.002 deg^2 values? In other words, is there something "special" about this resolution which makes the footprint and emissions inventories give reasonable results? 

Third, is there a reason to perform a summation instead of an integration? In principle these provide the same area under the curve, but in practice I've found integration to yield erroneous results.

Thank you for your help and your time,
Griffin

**Comment by tartanrunner25** [comment]
1) These assumptions include not accounting for the biosphere, which can be large depending on the location, and the background is assumed to be 400, which is a pretty loose approximation since this can vary dramatically based on the location and larger scale meteorology conditions.

2) The simulations resolution can be changed by changing the resolution of the footprint via the xres variable. Of course, if you change the resolution of the footprint, you will also need to change the resolution of the emissions in order to convolve the two together (essentially, we multiply them together to be left with dCO2 in ppm for each grid cell, before summing it to come up with the total contribution of CO2 from all emission sources). Usually what I do is I will re-run the STILT footprint at the resolution of the emission grid so the two match each other. That or you will need to regird the emissions to match the footprint grid spacing. 

3) This really means summation, i.e., once we multiple the footprint with the emissions, each grid cell has units of dCO2 per grid cell. Here, we just sum it up to come up with total dCO2 for the model domain.
